src/containers/guest-home-page/signup-form/SignupForm.jsx

```diff
@@ -70,6 +70,7 @@ const SignupForm = ({
       <Box sx={{ display: { md: 'block', lg: 'flex' }, gap: '15px' }}>
         <AppTextField
           autoFocus
+          data-testid='firstName-input'
```

Comment:
I don't see where you are using this `data-testid`. If you need to find this element, you can use `getByLabelText()`. Same for all changed fields.